k8s/base/daemon-config.json

```diff
@@ -0,0 +1,34 @@
+{
+    "anchor": {},
+    "http-api": {
+        "cors-allowed-origins": [
+            ".*"
```

Comment:
I think we need to be able to set this from config map.

Reply:
A configmap is generated from the file with the configmapgenerator.